common/src/main/java/org/keycloak/common/enums/AccountRestApiVersion.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.keycloak.common.enums;
+
+import java.util.Collections;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+/**
+ * @author Vaclav Muzikar <vmuzikar@redhat.com>
+ */
+public enum AccountRestApiVersion {
+    V1("v1");
+
+    public static final AccountRestApiVersion DEFAULT = V1;
+    private static final Map<String,AccountRestApiVersion> ENUM_MAP;
```

Comment:
You may use an [EnumMap](https://docs.oracle.com/javase/7/docs/api/java/util/EnumMap.html).

Reply:
I don't think we want a Map where Enum is the key here.